decisions/log/0027-footprint-fragment.md

```diff
@@ -0,0 +1,112 @@
+
+2. In section 6.8.2. Response Syntax, replace
+    ```
+    The host system upon not accepting the event SHOULD respond with an [=error response=] (see [[#api-error-responses]]).
+    ```
+    with
+    ```
+    If the host system accepts the event but cannot process it due to unavailability of the PF data for the     [=ProductFootprintFragment=]
+    in the request, it SHOULD return an [=error response=] (see [[#api-error-responses]]) with the [=error response code=]:     404 and appropriate [=error message=] indicating the unavailability of the PF data for the requested    [=ProductFootprintFragment=] value.
+    ```
+
+3. In the definition of `ProductFootprintFragment`, add
+    ```
+    It is RECOMMENDED to include at least the [=ProductIds=] in the PF Request Event request body.:
+    ```
+    followed by an example
+
+
+## Consequences
+
+1. As this ADR creates a non-breaking change, the Tech Specs Version number must be updated from 2.1.x to 2.2.y.
+
+## Appendix
+
+This appendix contains the text intended for the new section to be added to the Technical Specifications.
+
+### The General principles of Event processing
+
+1. A host system should strive to accept an event unless for the following reasons
+    1. event processing is not support by the host system
+    2. the syntax in the event is not supported (e.g. when the event failed “syntax checks” of the request host system)
+2. Whenever a host system’s `Event` endpoint returns an error response, no follow up response event (or error event) will be sent back to the original requester
+3. The requested host system will attempt to always send a response event to the requesting host system, **but the requesting host system should not rely on this behavior and retry after an appropriate amount of time by sending another new request event.**
+
+### Async Event Processing: Business Case 1 – Requesting Product Footprints
+
+#### Context and Assumptions
+
+- This business case is triggered by a data recipient if they wish to access a Product Footprint **they could not yet access through the `ListFootprints` API.**
+- The data recipient and data owner have a mutually agreed upon way to identify products
+
+#### Workflow
+
+1. The data recipient sends a `ProductFootprintFragment`  to the `Events Endpoint` of the data owner. The data recipient should always include 1 or more product ids in property `productIds`. The data recipient should limit each fragment to exactly 1 specific product (e.g. a specific type of apple instead of all apples that somebody is offering)
+    1. If the recipient is requesting a specific reporting period, it should include the reporting period accordingly (example …)
+    2. if the recipient is requesting a specific geography, it should include …
```

Comment:
This touches upon the core of using a `ProductFootprintFragment` for providing context (explaining what I am requesting). Lets assume I send fragment with reportingPeriod [1-1-2023 / 1-1-2024]. What does that mean?

-  I want a footprint with that exact reporting period?
- I want a footprint with that reporting period that falls within the given period?
- I want a footprint that roughly meets that period?
- that starts within, ends within, ...

Comparing it against the use of filters with `ListFootprints`, this way of providing context is not very straight forward. Basically, using a form like the fragment fits for 'equal' operators (productId = A1234) but not with most other operators.